src/savegame_json.cpp

```diff
@@ -3046,6 +3054,14 @@ void Creature::load( const JsonObject &jsin )
     }
     jsin.read( "values", values );
 
+    if( jsin.has_array( "damage_over_time_map" ) ) {
+        for( const JsonObject &obj : jsin.get_array( "damage_over_time_map" ) ) {
+            damage_over_time_data tmp_DoT;
+            tmp_DoT.load_DoT_data( obj );
+            add_damage_over_time( tmp_DoT );
+        }
+    }
```

Comment:
```suggestion
jsin.read( "damage_over_time_map", damage_over_time_data );
```
this is another spot that can be simplified but you must have a function named `deserialize`